deployments/gke/service.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: registry-backend
+#  Please uncomment the following lines to create an internal load balancer.
```

Comment:
I'm unfamiliar with the pros and cons of this, but I'm guessing that it's not a problem to leave this commented out.

Could this be reworded to make that more clear? Maybe instead of "Please uncomment..." it might say "If you uncomment the following lines, an internal load balancer will be created." followed by a sentence explaining the consequences of that - I guess something like "the load balancer will distribute requests among nodes that automatically scale up with load" (if that's what will happen) and if it seems appropriate, we might link to https://cloud.google.com/load-balancing/docs/internal

Reply:
Good catch! To make it less confusing, I created a separate config file for internal LB: deployments/gke/service-internal.yaml and added comments about internal LB on the top. Let me know if this makes thing more clear